src/demofile.cpp

```diff
@@ -147,7 +150,7 @@ bool CDemoFile::Open( const char *name )
 			return false;
 		}
 
-		fread( &m_DemoHeader, 1, sizeof( m_DemoHeader ), fp );
+		readed = fread( &m_DemoHeader, 1, sizeof( m_DemoHeader ), fp );
```

Comment:
`readed` sounds weird. is there a better variable name for this?

Reply:
@kaimallea `fread()` returns size of read bytes. So... `read_size` would be ok?